src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double area() {
+        return radius * radius * Math.PI;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, " + "area: " + area() + " sq. units, " + "radius: " + radius + " units, " + "color: " + getColor());
```

Comment:
Use `name()` instead of `toString()` for getting a String representation of enum constants. If `getColor()` is returning an enum, make sure it uses `name()`.